screens/EditBaniOrder.js

```diff
@@ -158,89 +203,51 @@ class Row extends React.Component {
   }
 
   render() {
-    const { data, active, nightMode, transliteration, fontFace } = this.props;
+    const { data, nightMode, transliteration, fontFace } = this.props;
 
     return (
-      <Animated.View
-        style={[
-          styles.row,
-          nightMode && { backgroundColor: "#000" },
-          this._style,
-        ]}>
-        {data.folder && (
-          <Image
-            source={require("../images/foldericon.png")}
-            style={styles.image}
-          />
-        )}
+      <Animated.View style={[styles.row, nightMode && { backgroundColor: "#000" }, this._style]}>
+        {data.folder && <Image source={require("../images/foldericon.png")} style={styles.image} />}
 
         <Text
           style={[
             { color: nightMode ? "#fff" : "#222222" },
```

Comment:
please try to remove hard coded string from here as well veerji

Reply:
fixed